home.html

```diff
@@ -0,0 +1,224 @@
+        </div>
+      </section>
+
+      <section class="second-sec">
+        <div class="main-container">
+          <div class="sec-sec-heading">
+            <h2>Main Program</h2>
+            <div class="underline1"><hr /></div>
+          </div>
+
+          <div class="flx2">
+            <div class="program-box1">
+              <div class="top-icon">
+                <i class="fa-solid fa-3x fa-microphone-lines"></i>
+              </div>
+              <div class="program-title">
+                <h2>Rap Music</h2>
+              </div>
+              <div class="program-description">
+                <p>
+                  Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                  Consequuntur, consectetur. Placeat et enim quae, ipsa
+                  perspiciatis optio sint eius quaerat?
+                </p>
+              </div>
+            </div>
+            <div class="program-box1">
+              <div class="top-icon">
+                <i class="fa-solid fa-3x fa-drum"></i>
+              </div>
+              <div class="program-title">
+                <h2>Live Show</h2>
+              </div>
+              <div class="program-description">
+                <p>
+                  Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                  Consequuntur, consectetur. Placeat et enim quae, ipsa
+                  perspiciatis optio sint eius quaerat?
+                </p>
+              </div>
+            </div>
+            <div class="program-box1">
+              <div class="top-icon">
+                <i class="fa-solid fa-3x fa-guitar"></i>
+              </div>
+              <div class="program-title">
+                <h2>R&B Music</h2>
+              </div>
+              <div class="program-description">
+                <p>
+                  Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                  Consequuntur, consectetur. Placeat et enim quae, ipsa
+                  perspiciatis optio sint eius quaerat?
+                </p>
+              </div>
+            </div>
+
+            <div class="program-box1">
+              <div class="top-icon">
+                <i class="fa-solid fa-3x fa-music"></i>
+              </div>
+              <div class="program-title">
+                <h2>Afro Beats</h2>
+              </div>
+              <div class="program-description">
+                <p>
+                  Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                  Consequuntur, consectetur. Placeat et enim quae, ipsa
+                  perspiciatis optio sint eius quaerat?
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="see-more">
+            <a href="#" class="see-more"> SEE THE WHOLE PROGRAM </a>
+          </div>
+          <button class="btn-prog" type="button">Join RG Music Festival</button>
+        </div>
+      </section>
+
+      <section class="third-sec">
```

Comment:
- [ ] Kindly check the design of this section, the design should match the design given except for content, I have done the comparison below for a better understanding, also increase the number of articles in this section to six:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/181730894-fa60d947-95ea-442f-8e51-3c64fa7c9db3.png) |  ![image](https://user-images.githubusercontent.com/61408860/181730957-cd637bc2-17dd-4a4b-a947-9222228e5f3d.png)